frontEnd/public/styles/roadmap.css

```diff
@@ -0,0 +1,8 @@
+#map {
+    margin: 100px auto;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    width: 500px;
```

Comment:
Try Use this instead:

#map {
    width: 100vw;
    height: 100vh;
}

Reply:
no need of absolute positioning and margin

Reply:
And just for some polishing, try including these too...

body{
    overflow-x: hidden;
}
h1{
    text-align: center;
}